docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+
+    ```
+    ---
+    date: 2023-12-31
+    ---
+
+    # Happy new years eve!
+
+    We hope you are all having fun and wish you all the best for the new year!
+    <!-- more -->
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua.
+    ```
+
+    Then, run `mkdocs serve` and point your web browser at
+    `http://localhost:8000/blog`.
+
+    Observe how the blog plugin automatically creates navigation elements for
+    the blog. Later on we will see how this can be integrated into navigation
+    for a site that contains not just a blog. Note the other elements of a blog
+    are present and only an extract is shown. When you select the "Continue
+    reading" link, you will get to the full blog post. Note how it has a URL
+    generated from the first-level heading.
+
+## 5. Post metadata
+
+### Drafts
+
+You may want to produce a draft of a blog post and work with it locally but
+exclude it from the build that you publish. You can add a field to the page
+header to indicate that a post is still in draft form.
+
+!!! example "Create a draft"
+
+    Create a second blog post in `docs/blogs/posts/draft.md` with the following
+    contents:
+
+    ```
+    ---
+    date: 2024-01-01
+    draft: true
+    ---
+
+    # Happy new year!
+
+    Happy 2024 to everyone. Wishing you all the best!
+    <!-- more -->
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua.
+    ```
+
+    Now, note how the draft appears on the index page but with a label that
+    indicates that it is a draft. When you run `mkdocs build`, the draft will
+    _not_ appear in the output:
+
+    === "MacOS/Linux"
+        ```
+        $ mkdocs build
+        $ ls site/blog
+        2023		archive		index.html
+        ```
+
+    === "Windows"
+        TODO
+
+    The first blog post for 2024 is not there yet because it is still in draft
+    stage. Remember to remove the `draft` setting in the header when it is time
+    to publish it.
+
+If you are using the [Insiders edition], you can also create
+a folder to keep your drafts in and use the [Meta plugin] to add the
+`draft` header setting to all the posts in that folder. We will cover the
+Meta plugin later on.
+
+[Meta plugin]: ../plugins/meta.md
+
+### Edits
+
+Sometimes, bloggers need to update a post. This might happen when they have made
+a mistake or when something changes they need to reflect in the post. To
+indicate you have edited a post, you can include an edit date in the page header:
+
+!!! example "Editing a post"
+
+    Make a change to your first blog post, then add an edit date to the header:
+
+    ```hl_lines="3"
+    ---
+    date: 2023-12-31
+    updated: 2024-01-02
+    ---
+    ```
+
+### Reading time
+
+To give the reader some idea of how long it might take them to read a post,
```

Comment:
IMHO this is far too verbose for what it describes. As a reader I would glaze over this block of text and just set reading time manually, missing that it is automatically set if omitted. Additionally it is not clear that it is automatically computed at all. The `readtime` should only ever be used as an override.